instrumentation/opentelemetry-instrumentation-confluent-kafka/src/opentelemetry/instrumentation/confluent_kafka/__init__.py

```diff
@@ -173,6 +173,9 @@ def __init__(self, consumer: Consumer, tracer: Tracer):
     def committed(self, partitions, timeout=-1):
         return self._consumer.committed(partitions, timeout)
 
+    def commit(self, message, asynchronous):
+        return self._consumer.commit(message=message, asynchronous=asynchronous)
```

Comment:
According to
https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#confluent_kafka.Consumer.commit
There is offsets param in the commit function, Is there a reason not to pass it?

Reply:
no reason other than I missed it! thanks for catching it, will update the PR

Reply:
Maybe it will be better to work with args and kwargs
so we won't assume the default values, and if the signature will change we still pass all the arguments